idl/group0.idl.hh

```diff
@@ -7,16 +7,23 @@
  */
 
 #include "idl/raft_storage.idl.hh"
+#include "gms/inet_address_serializer.hh"
 
 namespace service {
 
+struct discovery_peer {
+    raft::server_id id;
+    gms::inet_address ip;
```

Comment:
Continuing the thought from above: we could use bytes/string here as well, which would save us from creating new RPC verbs in the future if we change to URIs.

Reply:
Same comment applies to RPC. We use ip addresses in RPC everywhere.